WordPress/build.gradle

```diff
@@ -354,7 +354,7 @@ dependencies {
 
     implementation 'com.android.volley:volley:1.1.1'
     implementation 'com.google.firebase:firebase-messaging:21.1.0'
-    implementation 'com.google.android.gms:play-services-auth:18.1.0'
+    implementation 'com.google.android.gms:play-services-auth:20.3.0'
```

Comment:
> Updates play services auth library from 18.1.0 to 20.3.0

Nice catch! 🥇 

I am seeing some transitive dependencies got updated due to this `play-services-auth` update, which is expectable, but it makes me wonder if we should take this opportunity and at least update [WordPress-Login-Flow-Android](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android) library and its such dependency to `20.3.0` as well, that is, at some point (see [here](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/blob/trunk/WordPressLoginFlow/build.gradle#L56)). 🤔

PS: I don't think this is a blocker to this PR, just something that we might consider doing as a follow-up later on maybe, just so these dependencies are aligned. I am also seeing WCAndroid is already on `20.2.0`, which makes me less worry about this update for WPAndroid (see [here](https://github.com/woocommerce/woocommerce-android/blob/trunk/WooCommerce/build.gradle#L189)). 😅

Reply:
@ParaskP7 Great suggestion. I was afraid I might break something in `WCAndroid` if I updated the auth library in FluxC.Thats why I didn't update the auth library on FluxC, thanks for confirming that WC android already uses `20.2.0`. 🙌🏼 


I can definitely update auth library version in FluxC library and then point this PR to that update or I can take up updating dependencies in the FluxC later on. I wanted to limit the testing scope as well as the changes of this PR to minimal. So I prefer that we can take up updating the dependencies of FluxC a bit later on. WDYT? 🤔

Reply:
@ParaskP7 Oops my mistake. You are right. Login Library. Will update the comments.

Reply:
Awesome, deal, thank you @AjeshRPai ! 🥇